.github/workflows/static-checking.yml

```diff
@@ -27,7 +27,7 @@ jobs:
       - name: Install Requirements
         run: |
           python -m pip install --upgrade pip
-          pip install -r requirements-dev.txt
+          pip install -U -r requirements-dev.txt
       - name: CloudFormation Lint - Base
         run: cfn-lint -t cloudformation/base.yaml
       - name: CloudFormation Lint - Databases
```

Comment:
Should we limit the `python-version` for static-checking.yml to 3.7 as well?

Reply:
Yes.
Do you already did it in the governed tables branch, right? Should I replicate it here?

Reply:
Done!